zerver/tests/test_events.py

```diff
@@ -2898,6 +2898,7 @@ def do_set_realm_property_test(self, name: str) -> None:
             video_chat_provider=[
                 Realm.VIDEO_CHAT_PROVIDERS["jitsi_meet"]["id"],
             ],
+            jitsi_server_url=["https://example1.jit.si", "https://example2.jit.si"],
```

Comment:
Let's do `jitsi1.example.com`, and `jitsi2.example.com` -- we want to always use URLs in the official RFC `example.com` domain, not those of third parties, for this sort of thing.